rustls/src/crypto/ring.rs

```diff
@@ -1,7 +1,12 @@
+use std::fmt;
```

Comment:
Nit: local style is to put `std` imports below crate and external import blocks.